Author: Adam Buchbinder <adam.buchbinder@gmail.com>
Description:
 UTF-32BE text is detected by the presence of the Byte Order Mark, in UTF-32BE
 encoding. The stock version of the BOM is incorrect; it should read 00 00 FE
 FF, according to the Unicode FAQ: http://unicode.org/faq/utf_bom.html#bom4
 (Closes: #513526).

diff -Naurp file.orig/magic/Magdir/unicode file/magic/Magdir/unicode
--- file.orig/magic/Magdir/unicode	2009-03-21 09:20:03.000000000 +0000
+++ file/magic/Magdir/unicode	2009-03-21 17:02:09.000000000 +0000
@@ -9,6 +9,6 @@
 0	string	+/v+			Unicode text, UTF-7
 0	string	+/v/			Unicode text, UTF-7
 0	string	\335\163\146\163	Unicode text, UTF-8-EBCDIC
-0	string	\376\377\000\000	Unicode text, UTF-32, big-endian
+0	string	\000\000\376\377	Unicode text, UTF-32, big-endian
 0	string	\377\376\000\000	Unicode text, UTF-32, little-endian
 0	string	\016\376\377		Unicode text, SCSU (Standard Compression Scheme for Unicode)
